fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/domain/LoanAccountDomainServiceJpa.java

```diff
@@ -227,7 +230,7 @@ public LoanTransaction makeRepayment(final LoanTransactionType repaymentTransact
         }
 
         postJournalEntries(loan, existingTransactionIds, existingReversedTransactionIds, isAccountTransfer, isLoanToLoanTransfer);
-
+        loanAccrualTransactionBusinessEventService.raiseBusinessEventForAccrualTransactions(loan, existingTransactionIds);
```

Comment:
Please raise the event after the "recalculateAccruals" method which might created new Accrual transaction as well.